test/lib.encryption.test.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import { generateKey, encrypt, decrypt } from '../server/lib/encryption';
+
+describe('lib.encryption', () => {
+  it('it encrypts and decrypts ok', () => {
+    const message = 'OpenCollective Rules';
+    const buff = Buffer.from(message);
+    const key = generateKey();
+
+    const encrypted = encrypt(buff, key);
+
+    const result = decrypt(encrypted, key);
+
+    expect(result).to.eq(message);
```

Comment:
I would just add this to ensure the message is indeed encrypted (otherwise early returns in encrypt+decrypt could bypass the test):

```es6
expect(encrypted).to.not.eq(message);
```

Reply:
done.